TS3Client/Full/Ts3FullClient.cs

```diff
@@ -186,7 +187,17 @@ private void PacketEvent(ConnectionContext ctx, ref Packet<S2C> packet)
 				{
 				case PacketType.Command:
 				case PacketType.CommandLow:
-					LogCmd.ConditionalDebug("[I] {0}", Util.Encoder.GetString(packet.Data));
+					var str = Util.Encoder.GetString(packet.Data);
+					LogCmdIn.Trace("[I] {0}", str);
+					if (str.StartsWith("notify")) {
+							if (str != lastnotify)
+							{
+								lastnotify = str;
+								LogCmdServer.Debug(str);
+							} else {
+								LogCmdServer.Debug("+1");
+							}
+						}
```

Comment:
This block is a horrible hack; doesn't catch all notifications, produces allocations, and overhead. And doesn't even save that much readability as in a normal console where you can change the `+1` to a `+2`

Reply:
It kinda works for me, i would prefer to have it logged through a plugin to a Sql(ite) database though (https://github.com/Bluscream/TS3AudioBotPlugins/blob/da3dacfe4363d147b88f7e7249295e692ea91632/ServerLog/Plugin.cs). My attempt with litedb wasn't really successfull either (https://github.com/Bluscream/TS3AudioBotPlugins/blob/d1cd1442e8792b948b3d15a1f9e95083d206a73c/ServerLog/Plugin.cs).